### PR TITLE
Add ROLLO

### PR DESCRIPTION
**Pull request by TomerHawk**

### CODE REVIEW

File: encryption/rollo/rollo.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - Philippe Gaborit
+  - Adrien Hauteville
+  - Ayoub Otmani
+  - Olivier Ruatta
+  - Jean-Pierre Tillich
+  - Gilles Zémor
+links:
+  - 'https://pqc-rollo.org/'
+sources:
+  - 'https://pqc-rollo.org/'
```
Comment (dingens) [inline encryption/rollo/rollo.yaml:25]: can you link to a specific pdf? (or several)

File: encryption/rollo/rollo-I/param/rollo-I-128.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: ROLLO-I-128
+failure probability: -28
+failure probability comment: 2^-28
+sizes:
+  pk: 696
+  sk: 40
+  ct|sig: 696
+  comment: 'SessionKey: 64'
```
Comment (dingens) [inline encryption/rollo/rollo-I/param/rollo-I-128.yaml:8]: this is `msg`

File: encryption/rollo/rollo-I/param/rollo-I-128.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: ROLLO-I-128
+failure probability: -28
+failure probability comment: 2^-28
```
Comment (dingens) [inline encryption/rollo/rollo-I/param/rollo-I-128.yaml:3]: this is the same by convention, comment is duplicate

File: encryption/rollo/rollo.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+stateful: false
+name: 'ROLLO '
```
Comment (dingens) [inline encryption/rollo/rollo.yaml:2]: let's remove that space shall we ;)

File: encryption/rollo/rollo-II/rollo-II.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+name: ROLLO -II
```
Comment (dingens) [inline encryption/rollo/rollo-II/rollo-II.yaml:1]: better without space right?

File: encryption/rollo/rollo-I/impl/avx.yaml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+name: AVX2
+platform: AVX2
```
Comment (dingens) [inline encryption/rollo/rollo-I/impl/avx.yaml:2]: see hqc

File: encryption/rollo/rollo-II/bench/avx_rollo-II-128_opt.yaml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+platform: AVX2
```
Comment (dingens) [inline encryption/rollo/rollo-II/bench/avx_rollo-II-128_opt.yaml:1]: this should be processor (speed), compiler, etc.

Comment (dingens) [reply encryption/rollo/rollo-II/bench/avx_rollo-II-128_opt.yaml:1]: please see https://github.com/cryptoeng/pqdb/pull/124#discussion_r456401690 about the filenames

File: encryption/rollo/rollo-II/rollo-II.yaml [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+comment: LOCKER became ROLLO-II
+type: KTM
+security notion: IND-CCA
+security notion comment: IND-CCA2
```
Comment (dingens) [inline encryption/rollo/rollo-II/rollo-II.yaml:5]: also unneccessary

File: encryption/rollo/rollo-I/param/rollo-I-128.yaml (hunk 2 above)
Comment (TomerHawk) [reply encryption/rollo/rollo-I/param/rollo-I-128.yaml:8]: About ROLLO III : "**Updates for April 22nd, 2020**
• For sake of simplicity we now only consider the ROLLO-I and ROLLO-II schemes"

Comment (TomerHawk) [reply encryption/rollo/rollo-I/param/rollo-I-128.yaml:8]: https://pqc-rollo.org/doc/rollo-specification_2020-04-21.pdf, page 3

**Comment by dingens** [comment]
I don't know if it makes much sense to include this after NIST has thrown it out... But in case: where do you have the quantum numbers from? (127, 191, 255)

**Comment by TomerHawk** [comment]
> I don't know if it makes much sense to include this after NIST has thrown it out... But in case: where do you have the quantum numbers from? (127, 191, 255)

I made a naive assumption that quantum computers would at least decrease the security level by 2. 
I missed to note this that was a mistake, I am sorry. 
In this case it would be useful to allow Strings in this field or to be allowed to leave this attribute empty since there was no given security level for QC.

**Comment by dingens** [comment]
> I made a naive assumption that quantum computers would at least decrease the security level by 2.

Ah ok, that does make sense, in general, however you missed a detail: it's not the security that is halved (2¹²⁸ → 2¹²⁷), but instead its reduced to its square root, or, in other words, the security level if measured in bits (the exponent) is halved: 2¹²⁸ → 2⁶⁴.

We need to decide whether we remove the requirement to enter data. But we'd need to rework the comparison tool then.

**Comment by dingens** [comment]
I've fixed a few minor things (first two commits). Also you seem to have used the wrong table in the benchmarks once, I fixed that. I've also fixed the security values (in a way that we can merge this now, but adding a comment so we remember to change it in case we change the schema)

Can you review my changes? If all is correct I'd say we can merge this.

**Pull request merged**